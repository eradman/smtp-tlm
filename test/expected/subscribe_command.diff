1a2
> eradman@eradman.com
