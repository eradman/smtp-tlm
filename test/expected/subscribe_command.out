10c10
< dev: /var/www/archive/rset-dev-2024.mbox,marc,ericshane@eradman.com
---
> dev: /var/www/archive/rset-dev-2024.mbox,marc,ericshane@eradman.com,eradman@eradman.com
smtpctl update table aliases
smtpctl update table members
entr -zpnd true
